packages/api-utils/lib/getSlug.js

```diff
@@ -13,5 +13,5 @@ const { slugify } = require("transliteration");
  * @returns {String} slugified string
  */
 export default function getSlug(slugString) {
-  return (typeof slugString === "string" && slugify(slugString)) || "";
+  return (typeof slugString === "string" && slugify(slugString, { allowedChars: "a-zA-Z0-9-/" })) || "";
```

Comment:
@tedraykov 

Actually coming back to this I am concerned about this change being a global one and not specific to tags and may result in some unexpected behavior. I wonder if we can add a second argument of allowedChars, so that the default behavior remains the same.